components/compliance-service/docs/adding_test_data.md

```diff
@@ -0,0 +1,59 @@
+# Adding nodes to populate the compliance reporting page
+
+from hab studio: 
+
+`chef_load_compliance_scans -N20` 
+
+# Adding nodes to populate the nodes and scan jobs pages
+
+_note: this should be run on your local system, not from within the studio_
+
+ensure you have `jq` installed
+
```

Comment:
should i add a quick note here along the lines of "if you haven't run it before or run it in a while, please run `CHEF_USERNAME=username scripts/get_secrets` first" ?

Reply:
I tried to run `CHEF_USERNAME=username scripts/get_secrets` and it needed vault. How do you install vault?

Reply:
`brew install vault` worked for me. Yeah, I think it would be nice to add these to the doc somewhere for people first time running this.

Reply:
https://github.com/chef/automate/blob/master/dev-docs/DEV_ENVIRONMENT.md#working-with-secrets 
ya i think running that part of it from the studio would also work?

Reply:
ill add some instructions

Reply:
Thank you. The steps worked for me.